charts/resources/broker/templates/broker-auth-secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.auth.secretName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  username: {{ .Values.auth.username | b64enc }}
+  password: {{ .Values.auth.password | b64enc }}
```

Comment:
Nit: there's no need to _pipe_ the value into `b64enc`. The following would work just as well, and imo, reads much cleaner:

```
username: {{ b64enc .Values.auth.username }}
password: {{ b64enc .Values.auth.password }}
```